healpy/visufunc.py

```diff
@@ -63,8 +63,6 @@
 
 from . import projaxes as PA
 import numpy as np
-import matplotlib
-import matplotlib.pyplot as plt
```

Comment:
Due to cd9818aeb74e879c267ca8dcae66eca449a8d1a4, you no longer need to move these imports from module to function scope.

Reply:
fixed in recent push